lambda/healthCheck.ts

```diff
@@ -0,0 +1,190 @@
+import {
+	MetricUnits,
+	Metrics,
+	logMetrics,
+} from '@aws-lambda-powertools/metrics'
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb'
+import { SSMClient } from '@aws-sdk/client-ssm'
+import middy from '@middy/core'
+import { fromEnv } from '@nordicsemiconductor/from-env'
+import mqtt from 'mqtt'
+import assert from 'node:assert/strict'
+import { WebSocket, type RawData } from 'ws'
+import { registerDevice } from '../devices/registerDevice.js'
+import { getSettings, type Settings } from '../nrfcloud/settings.js'
+import { logger } from './util/logger.js'
+
+const { DevicesTableName, stackName, amazonRootCA1, websocketUrl } = fromEnv({
+	DevicesTableName: 'DEVICES_TABLE_NAME',
+	stackName: 'STACK_NAME',
+	amazonRootCA1: 'AMAZON_ROOT_CA1',
+	websocketUrl: 'WEBSOCKET_URL',
+})(process.env)
+
+const log = logger('healthCheck')
+const db = new DynamoDBClient({})
+const ssm = new SSMClient({})
+
+const metrics = new Metrics({
+	namespace: 'hello-nrfcloud-backend',
+	serviceName: 'healthCheck',
+})
+
+const deviceId = 'health-check'
+const model = 'PCA20035+solar'
+const fingerprint = '29a.ch3ckr'
+const ts = Date.now()
+const gain = 3.12345
+const expectedMessage = {
+	'@context':
+		'https://github.com/hello-nrfcloud/proto/transformed/PCA20035%2Bsolar/gain',
+	ts,
+	mA: gain,
+}
+
+const accountDeviceSettings = await getSettings({
+	ssm,
+	stackName,
+})()
+
+const publishDeviceMessage =
+	(bridgeInfo: Settings) =>
+	async (deviceId: string, message: Record<string, unknown>) => {
+		await new Promise((resolve, reject) => {
+			const mqttClient = mqtt.connect({
+				host: bridgeInfo.mqttEndpoint,
```

Comment:
Device should publish message to nRF Cloud MQTT API. Because otherwise we are not testing that the bridge has correctly subscribed to the nRF MQTT API.